cfgov/tccp/enums.py

```diff
@@ -156,67 +156,67 @@ def make_choices(*choices):
 )
 
 
-StateChoices = make_choices(
-    "AA",
-    "AE",
-    "AK",
-    "AL",
-    "AP",
-    "AR",
-    "AS",
-    "AZ",
-    "CA",
-    "CO",
-    "CT",
-    "DC",
-    "DE",
-    "FL",
-    "FM",
-    "GA",
-    "GU",
-    "HI",
-    "IA",
-    "ID",
-    "IL",
-    "IN",
-    "KS",
-    "KY",
-    "LA",
-    "MA",
-    "MD",
-    "ME",
-    "MH",
-    "MI",
-    "MN",
-    "MO",
-    "MP",
-    "MS",
-    "MT",
-    "NC",
-    "ND",
-    "NE",
-    "NH",
-    "NJ",
-    "NM",
-    "NV",
-    "NY",
-    "OH",
-    "OK",
-    "OR",
-    "PA",
-    "PR",
-    "PW",
-    "RI",
-    "SC",
-    "SD",
-    "TN",
-    "TX",
-    "UT",
-    "VA",
-    "VI",
-    "VT",
-    "WA",
-    "WI",
-    "WV",
-    "WY",
-)
+StateChoices = [
+    ("AL", "Alabama"),
+    ("AK", "Alaska"),
+    ("AS", "American Samoa"),
+    ("AZ", "Arizona"),
+    ("AR", "Arkansas"),
+    ("AA", "Armed Forces - Americas"),
+    ("AE", "Armed Forces - Europe"),
+    ("AP", "Armed Forces - Pacific"),
+    ("CA", "California"),
+    ("CO", "Colorado"),
+    ("CT", "Connecticut"),
+    ("DE", "Delaware"),
+    ("DC", "District of Columbia"),
```

Comment:
A question I have no answer to: should "DC" be "District of Columbia" as it is here or "Washington, D.C."? I couldn't find any solid best practices. Let's leave it like this for now, and I'll do a little poll among our DC colleagues and see what they expect.